lib/fs.dart

```diff
@@ -66,6 +66,29 @@ class FS {
         .event(name: name, properties: properties);
   }
 
+  /// Create a network event in the Dev Log in playback.
+  ///
+  /// [method] should be a valid HTTP method, such as GET, POST, PUT, or DELETE.
+  /// [requestSize] and [responseSize] are measured in bytes.
+  static Future<void> networkEvent({
+    required String url,
+    required String method,
+    int statusCode = 0,
+    int durationMs = 0,
+    int requestSize = 0,
+    int responseSize = 0,
+  }) =>
+      FullstoryFlutterPlatform.instance.captureEvent({
+        // This event type is assigned to network events in host SDKs
+        "eventType": 1,
```

Comment:
Could we make this a constant or an enum value instead of a "magic number"?

Reply:
Done!

Reply:
I guess can add these as needed, the analyzer apparently doesn't like having unused values.